web/rules/scripts/build_rules.py

```diff
@@ -0,0 +1,158 @@
+
+    html_content = f"""
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>{rule.name}</title>
+        <link rel="icon"       href="./img/favicon.ico" type="image/x-icon">
+        <link rel="stylesheet" href="./css/bootstrap-5.3.3.min.css">
+        <link rel="stylesheet" type="text/css" href="./css/styles.css">
+        <script src="./js/jquery-3.5.1.slim.min.js"></script>
+        <script src="./js/bootstrap-5.3.3.bundle.min.js"></script>
+        <script defer src="https://cloud.umami.is/script.js" data-website-id="0bb8ff9e-fbcc-4ee2-9f9f-b337a2e8cc7f"></script>
+        <style>
+        :root {{
+            /* from the icon */
+            --capa-blue: #2593d7;
+            --capa-blue-darker: #1d74aa;
+
+            --bs-primary: var(--capa-blue);
+            --bs-primary-rgb: var(--capa-blue);
+        }}
+
+        a:not(.btn) {{
+            color: var(--capa-blue);
+            text-decoration: none;
+        }}
+
+        a:not(.btn):hover {{
+            text-decoration: underline;
+            text-decoration-color: var(--capa-blue) !important;
+        }}
+
+        .rule-content .highlight pre {{
+            overflow: visible;
+        }}
+        </style>
+    </head>
+    <body>
+        <header 
+                class="d-flex flex-wrap justify-content-center py-1 mb-4 border-bottom fixed-top"
+                style="background-color: rgba(255,255,255,0.95);
+                box-shadow: 0 0.5rem 1rem rgba(0,0,0,0.05),inset 0 -1px 0 rgba(0,0,0,0.15);"
+            >
+            <a href="/" class="d-flex align-items-center mb-3 mb-md-0 me-md-auto">
+                <img src="./img/logo.png" height=48 />
+            </a>
+
+            <ul class="nav nav-pills">
+                <li class="nav-item d-flex align-items-center"><a href="/capa/#rules"    class="nav-link text-dark">Rules</a></li>
+                <li class="nav-item d-flex align-items-center"><a href="/capa/#examples" class="nav-link text-dark">Examples</a></li>
+                <li class="nav-item d-flex align-items-center"><a href="/capa/#download" class="nav-link text-dark">Download</a></li>
+            </ul>
+        </header>
+        
+        <div class="container d-flex justify-content-center" style="margin-top: 4rem !important;">
+            <div style="max-width: 650px;">
+                <p class="lead mb-0 text-secondary">
+                    <a href="{ns_link}" class="text-secondary">
+                        {namespace}
+                    </a>
+                </p>
+                <h1 class="display-6">{rule.name}</h1>
+                
+                <ul style="display: block; position: relative; float: right; height: 0px;" class="mt-4">
+                    <li><a href="{gh_link}">edit on GitHub</a></li>
+                    <li><a href="{vt_link}">search on VirusTotal</a></li>

```

Comment:
Ideally, external links should be opened in a new tab (`target="_blank"`) and use `rel="noopener noreferrer"`

Reply:
Interesting, my expectations are that a typical click opens in the same tab, and a middle click opens in a new tab, and that this customization is best left up to the user.